mewarpx/examples/capacitive_discharge.py

```diff
@@ -0,0 +1,226 @@
+"""
+Monte-Carlo Collision script benchmark against case 1 results from
+Turner et al. (2013) - https://doi.org/10.1063/1.4775084
+"""
+from pywarpx import picmi
+import pywarpx
+
+import numpy as np
+import time
+
+import shutil
+import yt
+
+from minerva import util as minutil
+
+constants = picmi.constants
+
+##########################
+# physics parameters
+##########################
+
+D_CA = 0.067 # m
+
+N_INERT = 9.64e20 # m^-3
+T_INERT = 300.0 # K
+
+FREQ = 13.56e6 # MHz
+
+VOLTAGE = 450.0
+
+M_ION = 6.67e-27 # kg
+
+PLASMA_DENSITY = 2.56e14 # m^-3
+T_ELEC = 30000.0 # K
+
+##########################
+# numerics parameters
+##########################
+
+# --- Grid
+nx = 128
+ny = 16
+
+xmin = 0.0
+ymin = 0.0
+xmax = D_CA
+ymax = D_CA / nx * ny
+
+number_per_cell_each_dim = [32, 16]
+
+DT = 1.0 / (400 * FREQ)
+
+# Total simulation time in seconds
+TOTAL_TIME = 10.0 * DT # 1280 / FREQ
+# Time (in seconds) between diagnostic evaluations
+DIAG_INTERVAL = 2.0 * DT # 32 / FREQ
+
+# --- Number of time steps
+max_steps = int(TOTAL_TIME / DT)
+diag_steps = int(DIAG_INTERVAL / DT)
+diagnostic_intervals = "::%i" % diag_steps #"%i:" % (max_steps - diag_steps + 1)
+
+print('Setting up simulation with')
+print('  dt = %.3e s' % DT)
+print('  Total time = %.3e s (%i timesteps)' % (TOTAL_TIME, max_steps))
+print('  Diag time = %.3e s (%i timesteps)' % (DIAG_INTERVAL, diag_steps))
+
+##########################
+# physics components
+##########################
+
+v_rms_elec = np.sqrt(minutil.kb_J * T_ELEC / minutil.m_e)
```

Comment:
@roelof-groenewald Please replace with WarpX / non-minutil constants and remove that import.